### PR TITLE
Adjust docs to Material 9.0.x theme

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
Adjusted to Material 9.0.x theme
Added view and edit buttons
Added link to Percona icon to lead to Percona docs homepage

modified: mkdocs-base.yml
modified: mkdocs.yml
new file: overrides/.icons/edit_page.png
new file: overrides/.icons/file-document-edit-outline.svg
new file: overrides/.icons/file-eye-outline.svg
modified: overrides/main.html
new file: overrides/partials/copyright.html
new file: overrides/partials/header.html
new file: overrides/partials/source-file.html

### CODE REVIEW

**Comment by atymchuk** [comment]
@alina-derkach-oaza May we ask you to sign the CLA please?

File: overrides/main.html [hunk 1]
```diff
@@ -1,30 +1,85 @@
...
-{{ super() }}
-
-<img src="https://tracking.g2crowd.com/attribution_tracking/conversions/1007137.gif?e=" width="1" height="1" />
-
-<noscript>
-  <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K48FGP2" height="0" width="0" style="display:none;visibility:hidden"></iframe>
-</noscript>
-{% endblock %}
+{#
+  MkDocs template for Netlify builds to customize docs layout
```
Comment (atymchuk) [inline overrides/main.html:2]:
```suggestion
  MkDocs template to customize the layout 
```

File: overrides/partials/copyright.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{#-
+  This file was automatically generated - do not edit
```
Comment (atymchuk) [inline overrides/partials/copyright.html:2]: By whom? Does the call not to edit hold true? It's edited as far as I can see.

**Comment by atymchuk** [comment]
Can we get a preview please?

**Review by atymchuk — DISMISSED** [review]

File: mkdocs.yml [hunk 3]
```diff
@@ -3,18 +3,24 @@ INHERIT: mkdocs-base.yml
...
+
+
+features:
+  - search.highlight
+  - navigation.top
+  - navigation.tracking 
+  - content.tabs.link
+  - content.action.edit
+  - content.action.view
+  - content.code.copy
```
[the remark this answers is not in the document]
Comment (alina-derkach-oaza) [reply mkdocs.yml]: updated

File: overrides/main.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (alina-derkach-oaza) [reply overrides/main.html]: removed

File: overrides/main.html [hunk 4]
```diff
@@ -1,30 +1,85 @@
...
-
-<img src="https://tracking.g2crowd.com/attribution_tracking/conversions/1007137.gif?e=" width="1" height="1" />
-
-<noscript>
-  <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-K48FGP2" height="0" width="0" style="display:none;visibility:hidden"></iframe>
-</noscript>
-{% endblock %}
+{#
+  MkDocs template for Netlify builds to customize docs layout 
+  by adding marketing-requested elements
```
[the remark this answers is not in the document]
Comment (alina-derkach-oaza) [reply overrides/main.html]: Updated

File: overrides/partials/copyright.html (hunk 2 above)
Comment (alina-derkach-oaza) [reply overrides/partials/copyright.html]: I've removed this file. I've tested, and there is no need to add this file to PMM project

**Comment by alina-derkach-oaza** [comment]
> Can we have a preview please?

Here is the review https://test-pmm.onrender.com/setting-up/server/docker.html

> @alina-derkach-oaza May we ask you to sign the CLA please?

I've signed it, but the warning is still there. I cannot sign it once again.

**Review by rasika-chivate — APPROVED** [review]

**Comment by rasika-chivate** [comment]
> > > Can we have a preview please?
> > 
> > 
> > Here is the link to review the changes https://test-pmm.onrender.com/index.html
> > > @alina-derkach-oaza May we ask you to sign the CLA please?
> > 
> > 
> > I've signed it, but the warning is still there. I cannot sign it once again.
> 
> Could you please ask IT to help sort it out? Problem is we can't merge w/o the CLA.

https://percona.atlassian.net/servicedesk/customer/portal/2/HD-19105 has been raised.

**Comment by alina-derkach-oaza** [comment]
> Could you please ask IT to help sort it out? Problem is we can't merge w/o the CLA.

I've created the task for IT already.

**Review by atymchuk — APPROVED** [review]

**Comment by alina-derkach-oaza** [comment]
Closing this PR due to the CLA issue. Created https://github.com/percona/pmm-doc/pull/972 instead.

**Pull request closed without merging**